app/src/format.py

```diff
@@ -7,12 +7,7 @@
 
 import markdown
 
-from src.citations import (
-    CITATION_PATTERN,
-    merge_contiguous_cited_subsections,
-    move_citations_after_punctuation,
-    remap_citation_ids,
```

Comment:
Instead of calling this three functions, call `simplify_citation_numbers()` instead.